docs/sources/reference/commandline/cli.md

```diff
@@ -101,6 +101,7 @@ expect an integer, and they can only be specified once.
       --mtu=0                                Set the containers network MTU
       -p, --pidfile="/var/run/docker.pid"    Path to use for daemon PID file
       --registry-mirror=[]                   Preferred Docker registry mirror
+      -r, --root=""                          Set root user/uid remap option
```

Comment:
Maybe something that is closer to the term *user* ? `--user-root` ? I really dont know